packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -371,7 +371,7 @@ export abstract class ProjectBase extends cdk.Construct implements IProject {
 class ImportedProject extends ProjectBase {
   public readonly projectArn: string;
   public readonly projectName: string;
-  public readonly role?: iam.Role = undefined;
+  public readonly role: iam.Role = undefined;
```

Comment:
wait, what? `role` is not optional but equals `undefined`?

Reply:
This is not compiling, sometimes I push to move work between my desktop and laptop. Sorry if this introduces churn on your end.